targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -86,9 +94,6 @@ object TargetBody {
             targetPropsL,
             { case (n, t, _ /*ms*/ ) =>
               EditSiderealInput.name.set(n.value.some) >>> TargetQueries.UpdateSiderealTracking(t)
-            // >>> EditSiderealInput.magnitudes.set( ms.map(m =>
-            //        MagnitudeInput(m.value.toBigDecimal, m.band, none, m.system.some)
-            //      ))
```

Comment:
This was here waiting for the API to support it.

Reply:
Ah sorry I can restore it

Reply:
I should have put a comment